src/.nuget/NuGet.Config

```diff
@@ -3,14 +3,14 @@
   <packageRestore>
     <add key="enabled" value="True" />
   </packageRestore>
+  <packageSources>
```

Comment:
I didn't even notice there was a NuGet.Config in the root. I don't know why it's there.